crates/bevy_render/src/view/visibility/mod.rs

```diff
@@ -47,6 +47,55 @@ pub enum Visibility {
     Visible,
 }
 
+impl Visibility {
+    /// Returns `Visibility::Inherited` if true, else returns `Visibility::Hidden`.
+    #[inline]
+    pub fn from_bool_inherited(value: bool) -> Self {
+        if value {
+            Visibility::Inherited
+        } else {
+            Visibility::Hidden
+        }
+    }
+    /// Returns `true` if `self` is `Visibility::Visible` or `Visibility::Inherited`, else returns `false`.
+    #[inline]
+    pub fn to_bool_inherited(self) -> bool {
+        self != Visibility::Hidden
+    }
+    /// If `self` is `Visibility::Hidden`, sets it to `Visibility::Inherited`. Otherwise, sets it to `Visibility::Hidden`.
+    #[inline]
+    pub fn toggle_inherited(&mut self) {
+        *self = if *self == Visibility::Hidden {
+            Visibility::Inherited
+        } else {
+            Visibility::Hidden
+        };
+    }
+    /// Returns `Visibility::Visible` if true, else returns `Visibility::Hidden`.
+    #[inline]
+    pub fn from_bool_unconditional(value: bool) -> Self {
+        if value {
+            Visibility::Visible
+        } else {
+            Visibility::Hidden
+        }
+    }
+    /// Returns `true` if `self` is `Visibility::Visible`, else returns `false`.
+    #[inline]
+    pub fn to_bool_unconditional(self) -> bool {
```

Comment:
I don't think i'd ever be able to remember with confidence what `to_bool_unconditional` did without looking at the docs.
```pub fn is_unconditionally_visible(self) -> bool {``` would be better maybe? 
But neither seems as readable `visibility == Visibility::Visible` really.